Orange/widgets/data/owimpute.py

```diff
@@ -205,7 +205,7 @@ def __init__(self):
         self.defbggroup = bgroup
 
         bgroup.buttonClicked[int].connect(self.set_default_method)
-        box = group_box(self.tr("Individual attribute settings"),
+        box = group_box(self.tr("Individual Attribute settings"),
```

Comment:
Would it not make more sense to write "Individual Attribute Settings" then, all capitalized?